build.xml

```diff
@@ -390,6 +514,25 @@
       deployRoot="${source} sprint ${version}"
     />
   </target>
+  
+  <!-- SYNTAX: ant local2dev-specified-tests -Dversion=3.6 -Dsource=qa -DspecifiedTests=TestClass1,TestClass2   optional: -DcheckOnly=true -DrunAllTests=true -->
+  <target name="local2dev-specified-tests">
+     <!-- backup dev first -->
+     <antcall target="backup-dev">
+       <param name="version" value="${version}" />
+	  <param name="type" value="package" />
+	 </antcall>
+	 <antcall target="backup-dev">
+       <param name="version" value="${version}" />
+       <param name="type" value="destructiveChanges" />
+     </antcall>
+     <antcall target="copy-destructiveChanges">
+       <param name="version" value="${version}" />
+       <param name="source" value="${source}" />
+     </antcall>
+     <!-- deploy script -->
+	 <scepcificTest cmcuser="${cmc.dev.username}" cmcpass="${cmc.dev.password}" serverurl="${sandbox.serverurl}"/>

```

Comment:
Typo "scepcificTest".  Update names of username/password attributes per earlier comments.